frontend/src/store/ApplicationState.ts

```diff
@@ -49,3 +50,20 @@ export const selectedApplicationDisplayText = computed(() => {
         return '';
     }
 });
+
+export const filters = ref({
```

Comment:
What's this for, why is it in `ApplicationState.ts`? Or could you add comment for the purpose?

Reply:
this should have been removed.